.changeset/lazy-moons-double.md

```diff
@@ -0,0 +1,5 @@
+---
+"@near-js/biometric-ed25519": minor
```

Comment:
Probably also makes sense as just `patch` but given the combination of configuration and code changes it made sense to bump to `minor`.